### PR TITLE
Redux configuration

### PR DESCRIPTION
**Pull request by MuhammadHamza096**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
         "prettier/prettier": [
         "error",
         {
+            "endOfLine": "auto",
```
Comment (ShahirZain) [inline .eslintrc.json:36]: please discard the changes of eslintrc file

**Review by ShahirZain — CHANGES_REQUESTED** [review]

**Review by ShahirZain — APPROVED** [review]

File: src/App.js [hunk 2]
```diff
@@ -1,14 +1,28 @@
...
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
 import { withTheme } from '@material-ui/core/styles';
+import { Actions } from './redux/actions';
 import Button from './components/Button';
+const App = () => {
+  const dispatch = useDispatch();
+  const { isLogin } = useSelector((state) => {
+    return {
+      isLogin: state.login_logout.isLoggedIn,
```
Comment (SiddiquiMonis401) [inline src/App.js:10]: We can use de structuring assignment here ..

File: src/App.js [hunk 3]
```diff
@@ -1,14 +1,28 @@
...
 
-function App(props) {
-  console.log(props);
   return (
-    <>
-    <Button />
-    </>  
+    <div>
+      <h1>is Logged In: {isLogin.toString()} </h1>
+      <button onClick={() => changeLogin()} type="button">
```
Comment (SiddiquiMonis401) [inline src/App.js:21]: Can't we use changeLogin here directly ... Like ..
onClick={changeClick}

File: src/redux/actions/login-logout.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import { LOGIN, LOGOUT } from '../../scripts/constants';
```
Comment (SiddiquiMonis401) [inline src/redux/actions/login-logout.js:1]: I think we should not contain actionTypes in scripts directory it should be in redux directory

File: src/redux/reducers/login-logout.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import { LOGIN, LOGOUT } from '../../scripts/constants';
```
Comment (SiddiquiMonis401) [inline src/redux/reducers/login-logout.js:1]: Change the directory for import also here ...

**Review by SiddiquiMonis401 — APPROVED** [review]

**Pull request merged**